src/ray/util/internal/stream_redirection_utils.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright 2025 The Ray Authors.
```

Comment:
If we are going to start following the standard of an `internal/` directory, do you think it makes sense to push it to the top level like `ray/internal/util/` or prefer `ray/util/internal/` ?

Reply:
I prefer the later, that's what abseil does:
- For example, there's `internal` folder for `container`: https://github.com/abseil/abseil-cpp/tree/master/absl/container/internal
- And `internal` folder inside of `functional`: https://github.com/abseil/abseil-cpp/tree/master/absl/functional/internal